### DISCUSSION TITLE
[@types/nw.js] Looking for second owner

### DISCUSSION THREAD

**Discussion by ayushmanchhabra** [DefinitelyTyped/DefinitelyTyped#70077] [_Misc]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70052
A second owner would speed up review process.

**Comment by ayushmanchhabra** [comment]
@zkrige ?

**Reply by zkrige** [reply]
Sure. Happy to help out. If the workload becomes too much I'll step back

**Reply by ayushmanchhabra** [reply]
Great! Could you create a PR to add yourself as owner of @types/nw.js? You'll have to add your info to the [owners property in the package.json](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0f4cb0332118fb5c5b9edeff0cce9112b218a1eb/types/nw.js/package.json#L16).

**Comment by zkrige** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70375
